addons/website_links/tests/test_link_tracker.py

```diff
@@ -0,0 +1,93 @@
+from odoo.tests import TransactionCase, tagged
+
+
+@tagged('post_install', '-at_install')
+class TestLinkTracker(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.company_1 = cls.env['res.company'].create({
```

Comment:
Use the main company for company_1 maybe, that way you just have to create "the other company"